portal-impl/src/com/liferay/portal/util/PortalInstances.java

```diff
@@ -617,7 +616,7 @@ private PortalInstances() {
 		new CopyOnWriteArrayList<>();
 	private static final Set<String> _virtualHostsIgnoreHosts;
 	private static final Set<String> _virtualHostsIgnorePaths;
-	private static String[] _webIds;
```

Comment:
@yuhai Can you make it a map so that every company id is for sure associated with the corresponding web id?